ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -1,35 +1,16 @@
 # Install OMERO.web with a public user on localhost
+# Pass the command line 'upgrade-webapps' flag to upgrade webapps
```

Comment:
See my later comment, but if you want to keep it as it is there's a typo: `_` not `-`